lib/htmldump.js

```diff
@@ -130,6 +131,7 @@ function Dumper (articleChunkStream, options) {
     this.articles = [];
     this.waiters = [];
     this.done = false;
+    this.contentHost = 'http://' + options.prefix + '/api/rest_v1'
```

Comment:
prefix is used by itself though for the Host: header.  Or are you suggesting parsing the value of the new argument to get the hostname out, for that purpose?

Maybe there needs to be a host arg and a contentUrl arg?